utils/rpms/packaging/Dockerfile.ubuntu.20.04

```diff
@@ -1,11 +1,11 @@
 #
-# Copyright 2019-2020, Intel Corporation
+# Copyright 2019-2021, Intel Corporation
```

Comment:
Still no CACHEBUST in this file?

Reply:
That's an issue for daos-stack/packaging, which should in fact be a submodule here.